File: beecrowd/beecrowd-2862/becrowd-2862.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

var n = 0;
var p = 0;
console.log(n)
function addn() {
    readline.question('', num =>{
        n += num;
        readline.close();
    });
}

function addp() {
    readline.question('', num =>{
        p = num;
        readline.close();
    });
}

addn();

for (let index = 0; index < n; index++) {
    addp();
    if (p > 8000) {
        console.log("Mais de 8000!")
    } else {
        console.log("Inseto!")
    }
}

File: beecrowd/beecrowd-2862/becrowd-2862.rs

fn main() {
    let mut string = String::new();
    std::io::stdin().read_line(&mut string).unwrap();
    let num = string.trim().parse::<i32>().unwrap();
    for _i in 0..num {
        let mut string = String::new();
        std::io::stdin().read_line(&mut string).unwrap();
        let p = string.trim().parse::<i32>().unwrap();
        if p > 8000 {
            println!("Mais de 8000!");
        } else {
            println!("Inseto!");
        }
    }
}
